noxfile.py

```diff
@@ -19,7 +20,19 @@ def tests_impl(
     extras: str = "socks,secure,brotli",
     byte_string_comparisons: bool = True,
 ) -> None:
+    is_github_actions = "GITHUB_ACTIONS" in os.environ
```

Comment:
Since we likely want the same installs locally and in GitHub Actions I don't think we need to change our behavior whether we're in CI or not.